cfpb/cfgov-refresh
# Home page: fix card overflow issue on mobile

An issue with `flex-basis:0` elements inside `flex-direction:column` containers causes card content to overflow in IE10 and Webkit browsers on small screen sizes. Setting `flex-basis: auto` on cards below 600px fixes the problem. (GHE 49)

## Additions

- Sets `flex-basis` to  `auto` on cards in card groups below 600px (the original setting of `0`  on cards is needed to fix a width issue on desktop in IE10)

## Testing

1. Check out branch and run `yarn gulp styles`
2. Navigate to [home page](http://localhost:8000/?nhp=True) in Safari, reduce screen width to below 600px, and verify that content does not overflow cards



## Screenshots

### Before

<img width="473" alt="Screen Shot 2019-12-05 at 3 30 24 PM" src="https://user-images.githubusercontent.com/778171/70293012-c111f680-1794-11ea-8cbb-38a273cd17db.png">


### After

<img width="466" alt="Screen Shot 2019-12-05 at 3 30 06 PM" src="https://user-images.githubusercontent.com/778171/70293021-c53e1400-1794-11ea-98a2-1a09e0a8304f.png">

## Notes
I considered adjusting the original card `flex-basis:0` rule to only apply on larger screens, but went with the additional card group rule since the issue is caused by the container. I'm open to changing the approach, though

## Testing checklist

### Browsers
